static/src/styles/main.styl

```diff
@@ -1,41 +1,2 @@
-@import 'colours'
-@import 'fonts'
-
-// Constants
-$normalBorderRadius = 0.3em;
-$smallBorderRadius = 0.1em;
-
-body {
-  background: $lightBackground;
-  font-family: sans-serif;
-}
-
-.mono {
-  font-family: 'Space Mono', monospaced;
-}
-
-.feed {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: flex-start;
-  align-items: stretch;
-}
-
-.post {
-  background: $bright1;
-  padding: 1em;
-  border-radius: $normalBorderRadius;
-  margin: 0.4em;
-  flex: 30%;
-  height: 10em;
-  color: $lightText;
-}
-
-.post_type {
-  display: inline-block;
-  background: $lightBackground;
-  padding: 0.3em;
-  color: $bright1;
-  border-radius: $smallBorderRadius;
-}
\ No newline at end of file
+@require './core';
+@require './landing';
```

Comment:
Not a BEM thing, just for our own sanity.

Do you think requiring a set of CSS files would on a per-page basis would be better?